frontend/src/index.tsx

```diff
@@ -10,25 +14,58 @@ import {
 import { setContext } from "@apollo/client/link/context";
 
 import AUTHENTICATED_USER_KEY from "./constants/AuthConstants";
-import { getLocalStorageObjProperty } from "./utils/LocalStorageUtils";
+import {
+  getLocalStorageObjProperty,
+  setLocalStorageObjProperty,
+} from "./utils/LocalStorageUtils";
 // } graphql
 
 import "./index.css";
 import App from "./App";
 import reportWebVitals from "./reportWebVitals";
 
+const REFRESH_MUTATION = `
+  mutation {
```

Comment:
Mutation name here?

Reply:
Not using Apollo hooks here, is it still possible to use an operation name?

Reply:
It should be possible since operation names are part of graphql and not specific to apollo, however if it doesn't work it looks good to merge

Reply:
Ok all done, do you mind approving again? 🙂